completions/jump.completion.sh

```diff
@@ -0,0 +1,10 @@
+#! bash oh-my-bash.module
+
+if _omb_util_command_exists jump; then
+	_jump()
```

Comment:
Please follow the [naming convention](https://github.com/ohmybash/oh-my-bash/blob/master/CONTRIBUTING.md#naming-convention-of-functions-and-global-variables). I've fixed this in the other PR without explanation, so could you learn from the changes in the other PR?